p2p/discovery/bucket_test.go

```diff
@@ -0,0 +1,50 @@
+/**
+*  @file
+*  @copyright defined in go-seele/LICENSE
+ */
+package discovery
+
+import (
+	"github.com/magiconair/properties/assert"
+	"net"
+	"testing"
+)
+
+func Test_Bucket(t *testing.T) {
+	b := bucket{}
+
+	n := getNode("9000")
+
```

Comment:
I think the blank lines #15 and #17 can be removed:)

Reply:
delete #15